.github/workflows/create-release-experimental.yml

```diff
@@ -0,0 +1,75 @@
+name: Create new experimental release
+
+on:
+  # run when pushed to main is published,
+  # which creates a new tag
+  push:
```

Comment:
Some math: If we do push on each commit, the image we push is ~170mb compressed. acr starts charging for over 100GB consumed, and we merge roughly 2 commits a day (620 since 1 year ago), so that's a growth of ~300mb / day == 9GB/month.

We'll likely need to clean up old images eventually. I think that we can enable that feature in our ACR if we upgrade to premium.

How does a retention time for untagged images of 45d sound? We ship official releases probably around that often, and worst case if you're using a tagged image and it gets pruned you cna swap to a newer version of the image anyway?

Reply:
I think 45d is reasonable; we can change if anyone complains. 

The core scenarios we're enabling here are for dev & test, and 45d should be enough for most cases.

Reply:
Before we merge this let me fix our ACR settings then.

Reply:
k, I've updated our ACR to be premium tier with a 45d retention and also protected all of the ASO images in it.

Reply:
> Some math: If we do push on each commit, the image we push is ~170mb compressed. acr starts charging for over 100GB consumed, and we merge roughly 2 commits a day (620 since 1 year ago), so that's a growth of ~300mb / day == 9GB/month.

That's a good estimate, but in reality various layers of the images should be re-used and the net storage impact should be a lot lower, no?